### DISCUSSION TITLE
Is it possible to implement photoshop-style layers?

### DISCUSSION THREAD

**Discussion by Variann** [excalidraw/excalidraw#5335] [General]
I've recently started using Excalidraw with Obsidian and have found this tool to be super useful. I apologise if this topic has already been discussed, but I couldn't find any old discussions or feature requests for anything like this.

The current layering system is nice but it's nowhere near as powerful and useful as other drawing apps layering system.
Being able to change opacity of certain layers, hide certain layers and more is an incredibly useful feature.

**Comment by dwelle** [comment]
It's possible, but right now we are trying to keep Excalidraw UI simple and minimal.

**Reply by Variann** [reply]
That's too bad, I hope it will be added one day. Or at least have an option to enable it for more advanced users.
In my opinion Concepts App has implemented layers in a simple and minimal way.

**Comment by dr-kernel** [comment]
+1000
That's the thing I feel is really missing with exaclidraw. When I draw a story, it would be beneficial to put some elements on a specific layer such that you can use the layers to enable features of the story or design. It would be a powerful feature to truly enable the hybrid note taking style this software embodies..

For example:
My note has a lot of text from other documents next to diagrams. Perhaps I want to see the drawing with only the Graphics for a cleaner/simpler story, but for added context I can enable the text layer and now I can read details about each part of the story.

Also, A "note" card could also be beneficial; expanded pop-out that again adds context without cluttering the design, uses the layers backend. Belongs in its own request but used further evidence for the feature.
